### ISSUE TITLE
2.0.x operatorhub publishing is missing some role permissions

### ISSUE THREAD

**Issue by dmvolod** [infinispan/infinispan-operator#472]
@rigazilla looks like we are missing some role permissions (routes, ingress, etc.) data during operatorhub publications
https://github.com/operator-framework/community-operators/blob/master/community-operators/infinispan/2.0.1/infinispan-operator.v2.0.1.clusterserviceversion.yaml 

```
{"level":"info","ts":1595103744.0209982,"logger":"controller_infinispan","msg":"Reconciling Infinispan: update case","Request.Namespace":"ispn-test","Request.Name":"example-infinispan"}
E0718 20:22:24.024547       1 reflector.go:123] pkg/mod/k8s.io/client-go@v0.0.0-20191016111102-bec269661e48/tools/cache/reflector.go:96: Failed to list *v1.Route: routes.route.openshift.io is forbidden: User "system:serviceaccount:ispn-test:infinispan-operator" cannot list resource "routes" in API group "route.openshift.io" in the namespace "ispn-test"
```

Thanks the colleague of mine, @dsevost which is finding this during Quarkus demo preparation
https://github.com/e2e-openshift-demos/quarkus-demo/blob/b-step-03_4/jdg/datagrid-operator-setup.sh#L58

**Comment by rigazilla** [comment]
Thanks @dsevost and @dmvolod. I've issued 2 pr with the fix on the Hub:
https://github.com/operator-framework/community-operators/pull/2018
https://github.com/operator-framework/community-operators/pull/2019

**Comment by dmvolod** [comment]
Thanks, @rigazilla 
I'm closing as it has been fixed for operator v2.0.2
